src/types/model.ts

```diff
@@ -0,0 +1,187 @@
+import { DuosUserResponse } from './responseTypes';
+
+export type UserRoleName =
+  | 'Admin'
+  | 'Chairperson'
+  | 'Member'
+  | 'Researcher'
+  | 'Alumni'
+  | 'SigningOfficial'
+  | 'DataSubmitter'
+  | 'All';
```

Comment:
Technically, we don't have a back end role for `'All'`, but it might be helpful in this context if all means 
> Any role is allowed to access this resource